.gitlab-ci.yml

```diff
@@ -11,13 +11,14 @@ cache:
 build_wheel_debug_linux_python27:
   tags:
     - docker
+  image: docker:19.03.8
   services:
-    - docker:18.09-dind
-  image: docker:18.09
+    - docker:19.03.8-dind
   stage: build
   variables:
-    DOCKER_HOST: tcp://docker:2375/
+    DOCKER_HOST: tcp://docker:2375
     DOCKER_DRIVER: overlay2
+    DOCKER_TLS_CERTDIR: ""
```

Comment:
Is this line needed?

Reply:
for docker 19, at least from my observation, yes. Otherwise, docker will listen on 2376,  which is the port for HTTPS.